src/pkg/cli/compose/loader_test.go

```diff
@@ -67,13 +67,15 @@ func diff(actualRaw, goldenRaw string) error {
 func testRunCompose(t *testing.T, f func(t *testing.T, path string)) {
 	t.Helper()
 
-	composeRegex := regexp.MustCompile(`^(docker-)?compose.ya?ml$`)
+	composeRegex := regexp.MustCompile(`^(?i)(docker-)?compose.ya?ml$`)
```

Comment:
If you create a file called `COMPOSE.YAML`, `docker compose up` will load it. That's possibly because by default MacOS is case-insensitive (but case-preserving) so the `os.Stat` in compose-go will read the file regardless. This test tries to mimic that.